### DISCUSSION TITLE
Welcome to cli Discussions!

### DISCUSSION THREAD

**Discussion by tonicmuroq** [projecteru2/cli#49] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by CMGS** [comment]
https://github.com/golang-standards/project-layout/blob/master/README_zh.md

这里有对 Pkg 的看法
